extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItemsCollector.java

```diff
@@ -82,6 +82,18 @@ public CommentsInfoItem extract(CommentsInfoItemExtractor extractor) throws Pars
             addError(e);
         }
 
+        try {
+            resultItem.setReplies(extractor.getReplies());
+        } catch (Exception e) {
+            addError(e);
+        }
+
+        try{
+            resultItem.setIsReply(extractor.isAReply());
+        } catch (Exception e){
```

Comment:
Please adjust your code formatting to match the project / standard Java style.
```suggestion
        try {
            resultItem.setIsReply(extractor.isAReply());
        } catch (Exception e) {
```